app/src/hvr/cpp/native-lib.cpp

```diff
@@ -25,6 +26,8 @@
   JNIEXPORT return_type JNICALL              \
     Java_com_igalia_wolvic_PlatformActivity_##method_name
 
+#define LOGI(...) __android_log_print(ANDROID_LOG_INFO, "nativeLib", __VA_ARGS__)
```

Comment:
Shouldn't we use `LOGD` and correspondingly `ANDROID_LOG_DEBUG`?
The logs being added seem to be for debugging purposes, not very useful as general app info.

Reply:
Makes sense